desktop/assets/categories.coffee

```diff
@@ -13,7 +13,7 @@ routes =
 
   '''
   /tag
-  ''': require('../apps/tag/client.coffee').init
+  ''': require('../apps/tag/client.js').default.init
```

Comment:
By default yes, but you can [configure this in browserify](https://github.com/substack/node-browserify#browserifyfiles--opts) with `extensions`.